app/src/main/java/com/github/se/travelpouch/ui/dashboard/TravelActivity.kt

```diff
@@ -105,12 +105,19 @@ fun TravelActivitiesScreen(
       },
       bottomBar = {
         NavigationBar(modifier = Modifier.testTag("navigationBarTravel")) {
-          listOfDestinations.forEach {
+          listOfDestinations.forEach { destination ->
             NavigationBarItem(
-                onClick = {},
-                icon = { Icon(it.icon, contentDescription = null) },
+                onClick = {
+                  when (destination.title) {
+                    "Activities" -> navigationActions.navigateTo(Screen.TRAVEL_ACTIVITIES)
+                    "Map" ->
+                        navigationActions.navigateTo(
+                            Screen.TRAVEL_ACTIVITIES) // Todo: navigate to map screen
```

Comment:
I guess the two screens are similar because the map screen for activites has not been developped yet

Reply:
Yes, that is correct. We keep the ToDo for a later modification